docs/user/autoyast/reference.md

```diff
@@ -0,0 +1,402 @@
+| `debug`         | no        |                       |                                       |
+| `notification`  | no        |                       |                                       |
+| `param-list`    | no        |                       |                                       |
+| `rerun`         | no        |                       |                                       |
+
+### scripts/init-scripts[]
+
+This section is supported.
+
+| AutoYaST   | Supported | Agama                 | Notes |
+| ---------- | --------- | --------------------- | ----- |
+| `filename` | yes       | `scripts.init[].name` |       |
+| `location` | yes       | `scripts.init[].url`  |       |
+| `source`   | yes       | `scripts.init[].body` |       |
+| `rerun`    | no        |                       |       |
+
+## mail
+
+This section is not supported.
+
+## nfs
+
+This section is not supported.
+
+## nfs_server
+
+This section is not supported.
+
+## nis
+
+This section is not supported.
+
+## nis_server
+
+This section is not supported.
+
+## ntp-client
+
+This section is not supported.
+
+## printer
+
+This section is not supported.
+
+## proxy
+
+There are plans to support this section in the future. Set the proxy using the kernels' command line
+
+## report
+
+This section is not supported.
+
+## samba-client
+
+This section is not supported.
+
+## software
+
+This section is supported.
+
+| AutoYaST              | Supported | Agama                 | Notes        |
+| --------------------- | --------- | --------------------- | ------------ |
+| `do_online_update`    | no        |                       | No 2nd stage |
+| `install_recommended` | no        |                       |              |
+| `instsource`          | no        |                       |              |
+| `kernel`              | no        |                       |              |
+| `packages[]`          | planned   |                       |              |
+| `post-packages[]`     | no        |                       |              |
+| `patterns[]`          | yes       | `software.patterns[]` |              |
+| `products[]`          | yes       | `software.id`         |              |
+| `remove-packages[]`   | no        |                       |              |
+| `remove-patterns[]`   | no        |                       |              |
+| `remove-products[]`   | no        |                       |              |
+
+## sound
+
+This section is not supported.
+
+## squid
+
+This section is not supported.
+
+## ssh_import
+
+This section is not supported.
+
+## suse_register
+
+This section is supported.
+
+| AutoYaST                           | Supported | Agama                       | Notes                                   |
+| ---------------------------------- | --------- | --------------------------- | --------------------------------------- |
+| `do_registration`                  | yes       |                             | The while section is ignored if "false" |
```

Comment:
I guess this should be "the whole section" instead of "the while section". That being said, I still don't understand the explanation very well.

Reply:
Well, it refers to the whole `suse_register` section. After all, the table is in that section. However, I have changed the wording.